CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -0,0 +1,91 @@
+- Add a FlowExporter configuration toggle to antrea-agent for users to explicitly enable/disable flow exports. ([#5021](https://github.com/antrea-io/antrea/pull/5021), [@yuntanghsu])
+- Add OpenAPI schema for the AntreaAgentInfo and AntreaControllerInfo CRDs. ([#5206](https://github.com/antrea-io/antrea/pull/5206), [@ceclinux])
+- Add a validation schema for the matchLabels field of the ExternalIPPool CRD. ([#5284](https://github.com/antrea-io/antrea/pull/5284), [@tnqn])
+- Update short-names for AntreaNetworkPolicy as ANNP. ([#5081](https://github.com/antrea-io/antrea/pull/5081), [@qiyueyao])
+- Disable CGO for the antctl binary to avoid GLIBC dependency for cross-platform compatibility. ([#5240](https://github.com/antrea-io/antrea/pull/5240), [@XinShuYang])
+- Change the sftp image container registry to `projects.registry.vmware.com`. ([#5197](https://github.com/antrea-io/antrea/pull/5197), [@ceclinux])
+- Use a new API in ofnet to remove the unnecessary map to reduce agent memory usage. ([#5054](https://github.com/antrea-io/antrea/pull/5054), [@wenyingd])
+- Use syscall to query or operate network adapters on Windows to reduce operation delay. ([#4898](https://github.com/antrea-io/antrea/pull/4898), [@wenyingd] [@qiyueyao])
+- Update out-of-date audit logs docs for new log fields. ([#5199](https://github.com/antrea-io/antrea/pull/5199), [@cr7258])
+- Switched to structured logging and change verbosity of potentially misleading Info log in the Antrea NetworkPolicy reconciler. ([#5048](https://github.com/antrea-io/antrea/pull/5048), [@antoninbas])
+- Revert a change to serve the v1alpha2 version of the ClusterGroup CRD again for the consistent API promotion plan. ([#5277](https://github.com/antrea-io/antrea/pull/5277), [@GraysonWu])
+- Upgrade Open vSwitch to version 2.17.7. ([#5225](https://github.com/antrea-io/antrea/pull/5225), [@antoninbas])
+- Upgrade Windows Open vSwitch to version 3.0.5. ([#5120](https://github.com/antrea-io/antrea/pull/5120), [@wenyingd])
+- Upgrade ClickHouse go client to v2. ([#5020](https://github.com/antrea-io/antrea/pull/5020), [@heanlan])
+- Remove Antrea Octant plugin. ([#5049](https://github.com/antrea-io/antrea/pull/5049), [@antoninbas])
+
+### Fixed
+
+- Bump up `libOpenflow` and `ofnet` library versions to fix a PacketIn2 response parse error. ([#5154](https://github.com/antrea-io/antrea/pull/5154), [@wenyingd])
+- Bump up `libOpenflow` library to v0.12.1 to fix an antrea-agent crash issue when marshaling the IGMPv3 query packet. ([#5320](https://github.com/antrea-io/antrea/pull/5320), [@ceclinux])
+- Remove the associated stale conntrack entries when UDP Endpoints are removed to fix the UDP Service unavailable issue util a new Endpoint is selected. ([#5112](https://github.com/antrea-io/antrea/pull/5112), [@hongliangl])
```

Comment:
This could be in "Added". It's kind of a new capacity and is controlled by a feature gate.

Add support for removing the associated stale conntrack entries when UDP Endpoints are removed, with which UDP requests can be redirected to other Endpoints immediately rather than waiting for the conntrack entries to expire.